modDbImpl/test/org/aion/db/impl/DriverBaseTest.java

```diff
@@ -575,8 +579,9 @@ public void testDrop() {
     }
 
     @Test
+    /** This test is non-deterministic and may fail. If it does, re-run the test suite. */
```

Comment:
how do we know if failure is problematic or not?

Reply:
Personally, I don't see much value in it. It's a legacy test. I almost removed it.

Reply:
I got confirmation that the test can be ignored.